### PR TITLE
Added ordination plot function (#52)

### PR DESCRIPTION
**Pull request by sdhutchins**
- Added ordination plot function
- Added tests for oridination function
- Updated version to 0.1.2
- Added NEWS for the last 2 versions

### CODE REVIEW

File: R/ordination.R [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  plot <- plot + ggplot2::labs(color = color) + ggplot2::scale_color_manual(values = c("Control" = "#3288bd", "Stressed" = "#d53e4f"))
+
+  if (!is.null(title)) {
+    plot <- plot + ggplot2::ggtitle(title) + ggplot2::annotate("segment", x = Inf, xend = -Inf, y = Inf, yend = Inf, color = "black", lwd = 1) +
+      ggplot2::guides(color = ggplot2::guide_legend(override.aes = list(fill = NA)))
+  }
+
+  df <- phyloseq::plot_ordination(phyloseq_object, ord, justDF = TRUE, axes = 1:4)
+
+  if (save == TRUE) {
```
[the remark this answers is not in the document]
Comment (sdhutchins) [reply R/ordination.R]: I'm honestly not a fan of any of the `save` functions including what I have there. I'd rather just completely remove it tbh.

File: R/ordination.R [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+#' @export
+#' @rdname ordination_plot
+#' @seealso
+#'
+#' @importFrom metacoder as_phyloseq
+#' @importFrom phyloseq ordinate plot_ordination
+#' @importFrom ggplot2 element_text geom_point theme element_blank guide_legend guides ggtitle unit scale_fill_manual labs scale_color_manual
+ordination_plot <- function(obj, method = "PCoA", distance = "wunifrac", color = "TreatmentGroup", title = NULL, save = FALSE, only_data = FALSE) {
+  metacoder_object <- validate_MicrobiomeR_format(
+    obj = object_handler(obj),
```
[the remark this answers is not in the document]
Comment (sdhutchins) [reply R/ordination.R]: Can I leave this nested?

File: R/ordination.R (hunk 1 above)
[the remark this answers is not in the document]
Comment (grabear) [reply R/ordination.R]: That's alright.  I'll add one for ordination later.

File: R/ordination.R (hunk 2 above)
[the remark this answers is not in the document]
Comment (grabear) [reply R/ordination.R]: Sure.

File: R/ordination.R (hunk 1 above)
[the remark this answers is not in the document]
Comment (sdhutchins) [reply R/ordination.R]: @grabear I don't mind writing it up if you really prefer it as I'll need to further tweak that `ordination_plot` function for additional use cases. I'll likely work on that over the weekend a bit more.

**Comment by sdhutchins** [comment]
Helps address #16

**Pull request merged**